plugins/schemahero.yaml

```
@@ -0,0 +1,60 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: schemahero
+spec:
+  version: v0.9.0-alpha.1
+  platforms:
+  - selector:
+      matchLabels:
+        os: linux
+        arch: amd64
+    uri: https://github.com/schemahero/schemahero/releases/download/v0.9.0-alpha.1/kubectl-schemahero_linux_amd64.tar.gz
+    sha256: 78119f420f10fd9cc5d0d341cdc0404aea4ccc6d8da8646a0a34826e71d4dcd6
+    files:
+    - from: kubectl-schemahero
+      to: .
+    - from: LICENSE
+      to: .
+    bin: kubectl-schemahero
+  - selector:
+      matchLabels:
+        os: darwin
+        arch: amd64
+    uri: https://github.com/schemahero/schemahero/releases/download/v0.9.0-alpha.1/kubectl-schemahero_darwin_amd64.tar.gz
+    sha256: 79144272c3dc396b7b950bbd2a3ddd9045c49b248577dd00de7e1ed034b0f2ad
+    files:
+    - from: kubectl-schemahero
+      to: .
+    - from: LICENSE
+      to: .
+    bin: kubectl-schemahero
+  - selector:
+      matchLabels:
+        os: windows
+        arch: amd64
+    uri: https://github.com/schemahero/schemahero/releases/download/v0.9.0-alpha.1/kubectl-schemahero_windows_amd64.tar.gz
+    sha256: 5a293788484762fbc1c7d27737e40a39db9f7f162b482b651516c06a5d3a6479
+    files:
+    - from: kubectl-schemahero.exe
+      to: .
+    - from: LICENSE
+      to: .
+    bin: kubectl-schemahero.exe
+  shortDescription: Cloud native, declarative database schema migrations
```

```suggestion
  shortDescription: Declarative database schema migrations via YAMLs
```
It's unclear what cloud native even means. So I'd recommend to be more descriptive here.